const customEX = use('App/Exceptions/CustomException');

// throw new customEX();
